### ISSUE TITLE
upload file via stream component

### ISSUE THREAD

**Issue by lion01** [offiria/offiria#42]
hello,
I do not know if this is really a bug, I never managed to post files via stream component!
  even after increasing variable "upload_max_filesize", I always get a failure message

**Comment by sauanimal** [comment]
Maybe there is some kind of folder permission issue? I had the same problem when trying to change system logo or add my avatar. Unfortunately i dont remember the folders, maybe /tmp, /images/avatars or /media/avatars, /files!?

**Comment by lion01** [comment]
i have many joomla site installed in this server (wamp with php 5.4.3/MySQL 5.5.24)  and I have not had this kind of problems ...
you can check with me to confirm...

**Comment by JMSolo** [comment]
Did you add the upload and post size lines to your php.ini in both the root folder for offiria and your /administrator/ folder?

<pre code>post_max_size=120M
upload_max_filesize=120M</pre code>

**Comment by lion01** [comment]
Yes in media manager  Maximum Size 120M
and in php.ini

post_max_size=120M

upload_max_filesize=120M

for image file i get  a failure message like the picture

![file](https://f.cloud.github.com/assets/5323019/1103850/efff3690-18c9-11e3-9cf0-c87111bee65e.png)

for a zip file I GET a popup message "file is too large" and a failure message

**Comment by lion01** [comment]
for zip file size 5.5Mo !!!!!

**Comment by sauanimal** [comment]
Do you get something in log files about that, ie apache log or smth

**Comment by lion01** [comment]
This is what apache generates when I try to attach files

[Tue Sep 10 13:59:59 2013] [error] [client 127.0.0.1] File does not exist: C:/wamp/www/offiria/administrator/components/com_akeeba, referer: http://localhost/offiria/administrator/index.php
[Tue Sep 10 14:01:18 2013] [error] [client 127.0.0.1] File does not exist: C:/wamp/www/offiria/administrator/components/com_akeeba, referer: http://localhost/offiria/administrator/index.php?option=com_media
[Tue Sep 10 14:02:09 2013] [error] [client 127.0.0.1] File does not exist: C:/wamp/www/offiria/administrator/components/com_akeeba, referer: http://localhost/offiria/administrator/index.php?option=com_media

**Comment by uncore** [comment]
Same error. There some progress about this bug?

**Comment by xnovy** [comment]
hello
i had the same problem (upload file always failed) and for me helped  uncomment line in php.ini: extension=php_fileinfo.dll

**Comment by sathyasr80** [comment]
there is no php.ini file...

**Comment by JMSolo** [comment]
Yes, if you are having this error.. you need to rebuild Apache to include the Fileinfo extension to make use of these functions.  If you do not know how to do this, simply contact your host and request that you have fileinfo enabled.  It's a fairly simple fix.  

As per php.ini -- you can create your php.ini file by naming a blank file as such and uploading it to your server.  This really comes down to your host and how they have their server settings implemented.  Some hosts have a PHP Settings tab within cPanel where you can enable and disable a variety of settings and even add your own. If they allow users to locally overwrite the main php.ini file then it may or may not be necessary.  Depends on your level of security and any other additional settings that you are after.

I have used my own file in the past for open_basedirectory overrides. 

Again, contact your host.

I have included a snapshot of a simple php.ini file and some settings you could see within one.
![php_ini](https://f.cloud.github.com/assets/5215074/2171402/fb7b9488-9582-11e3-9ea3-f5ed13cc94dd.png)

**Comment by mjnkun** [comment]
have a same error. There some progress about this bug?

Always Failed attach file.

**Comment by oabbas** [comment]
hello
i had the same problem (upload file always failed) and for me helped uncomment line in php.ini: extension=php_fileinfo.dll

this solution works with me and solve the problem
